pkg/syncer/cnsoperator/controller/cnsnodevmbatchattachment/cnsnodevmbatchattachment_helper.go

```diff
@@ -763,6 +785,54 @@ func patchPVCAnnotations(ctx context.Context, k8sClient kubernetes.Interface,
 	return nil
 }
 
+// patchPVCBackingTypeAnnotation updates the BackingType annotation on the PVC.
+// This is used to cache the backing type so it doesn't need to be queried again on future attach operations.
+func patchPVCBackingTypeAnnotation(ctx context.Context, k8sClient kubernetes.Interface,
+	pvc *v1.PersistentVolumeClaim, backingType string) error {
+	log := logger.GetLogger(ctx)
+
+	patchAnnotations := make(map[string]interface{})
+	if pvc.Annotations != nil {
+		for k, v := range pvc.Annotations {
+			patchAnnotations[k] = v
+		}
+	}
+
+	log.Infof("Setting BackingType annotation %s=%s on PVC %s",
+		common.AnnKeyBackingDiskType, backingType, pvc.Name)
+	patchAnnotations[common.AnnKeyBackingDiskType] = backingType
+
+	// Build patch structure
+	patch := map[string]interface{}{
+		"metadata": map[string]interface{}{
+			"annotations": patchAnnotations,
+		},
+	}
+
+	patchBytes, err := json.Marshal(patch)
+	if err != nil {
+		log.Errorf("failed to marshal BackingType annotation for PVC %s. Err: %s", pvc.Name, err)
+		return fmt.Errorf("failed to marshal patch: %v", err)
+	}
+
+	log.Infof("Patching PVC %s with BackingType annotation", pvc.Name)
+
+	// Apply the patch
+	updatedpvc, err := k8sClient.CoreV1().PersistentVolumeClaims(pvc.Namespace).Patch(
+		ctx,
+		pvc.Name,
+		types.MergePatchType,
+		patchBytes,
+		metav1.PatchOptions{},
+	)
+	if err != nil {
+		log.Errorf("failed to patch PVC %s with BackingType annotation. Err: %s", pvc.Name, err)
+		return fmt.Errorf("failed to patch PVC %s: %v", pvc.Name, err)
+	}
+	log.Infof("Successfully patched PVC: %s with BackingType annotation %+v", pvc.Name, updatedpvc.Annotations)
```

Comment:
May be change to debug? Because it will a lot of annotations and flood our syncer logs.
Or we can just print the specific annotation type.